contrib/packs/actions/workflows/install.yaml

```diff
@@ -25,6 +25,12 @@
       ref: "packs.load"
       params:
         register: "{{register}}"
+      on-success: "reload-aliases"
```

Comment:
Done in c2d22599ba2789c5a87febe841eb52424df454e8.

If we are comfortable doing this in packs.install might as well handle in st2ctl.